### ISSUE TITLE
./tensorflow/lite/tools/make/download_dependencies.sh: line 59: 1: Usage: download_and_extract URL DIR

### ISSUE THREAD

**Issue by thelinuxmaniac** [tensorflow/tensorflow#36199]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2+deb10u2 (2019-11-11) x86_64 GNU/Linux
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): from source at https://github.com/tensorflow/tensorflow/archive/v2.1.0.tar.gz
- TensorFlow version: 2.1.0
- Python version: 3.7.3 (but not relevant to this issue as I am only compiling tf lite using make)
- Installed using virtualenv? pip? conda?: n/a
- Bazel version (if compiling from source): n/a
- GCC/Compiler version (if compiling from source): gcc (Debian 8.3.0-6) 8.3.0
- CUDA/cuDNN version: n/a
- GPU model and memory: n/a

**Describe the problem**

```
$ wget https://github.com/tensorflow/tensorflow/archive/v2.1.0.tar.gz
$ tar -zxvf v2.1.0.tar.gz 
$ cd tensorflow-2.1.0/
$ ./tensorflow/lite/tools/make/download_dependencies.sh 
./tensorflow/lite/tools/make/download_dependencies.sh: line 59: 1: Usage: download_and_extract URL DIR
```

**Any other info / logs**
It seems that the `EIGEN_URL` variable is not being set correctly because the following and `grep -o` fails in the following command: `EIGEN_URL="$(grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v mirror.tensorflow | head -n1)"`. The eigen URL should be updated to something like `https://storage.googleapis.com/mirror.tensorflow.org/gitlab.com/libeigen/eigen/`.

```
bash -x ./tensorflow/lite/tools/make/download_dependencies.sh 
+ set -e
+++ dirname ./tensorflow/lite/tools/make/download_dependencies.sh
++ cd ./tensorflow/lite/tools/make
++ pwd
+ SCRIPT_DIR=/home/tlm/_tmp/imfind/tf/tensorflow-2.1.0/tensorflow/lite/tools/make
+ cd /home/tlm/_tmp/imfind/tf/tensorflow-2.1.0/tensorflow/lite/tools/make/../../../..
+ DOWNLOADS_DIR=tensorflow/lite/tools/make/downloads
+ BZL_FILE_PATH=tensorflow/workspace.bzl
+ '[' '!' -f tensorflow/workspace.bzl ']'
++ grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' tensorflow/workspace.bzl
++ grep -v mirror.tensorflow
++ head -n1
+ EIGEN_URL=
++ grep -o 'https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/gemmlowp/.*zip' tensorflow/workspace.bzl
++ head -n1
+ GEMMLOWP_URL=https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/gemmlowp/archive/12fed0cd7cfcd9e169bf1925bc3a7a58725fdcc3.zip
+ GOOGLETEST_URL=https://github.com/google/googletest/archive/release-1.8.0.tar.gz
++ grep -o 'https://github.com/abseil/abseil-cpp/.*tar.gz' tensorflow/workspace.bzl
++ head -n1
+ ABSL_URL=https://github.com/abseil/abseil-cpp/archive/43ef2148c0936ebf7cb4be6b19927a9d9d145b8f.tar.gz
+ NEON_2_SSE_URL=https://github.com/intel/ARM_NEON_2_x86_SSE/archive/master.zip
+ FARMHASH_URL=https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/farmhash/archive/816a4ae622e964763ca0862d9dbd19324a1eaf45.tar.gz
+ FLATBUFFERS_URL=https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/flatbuffers/archive/v1.11.0.tar.gz
+ FFT2D_URL=https://storage.googleapis.com/mirror.tensorflow.org/www.kurims.kyoto-u.ac.jp/~ooura/fft2d.tgz
+ download_and_extract '' tensorflow/lite/tools/make/downloads/eigen
+ local 'usage=Usage: download_and_extract URL DIR'
./tensorflow/lite/tools/make/download_dependencies.sh: line 59: 1: Usage: download_and_extract URL DIR
```

**Comment by thelinuxmaniac** [comment]
Updating the `tensorflow/lite/tools/make/download_dependencies.sh` file as follows fixes this issue:
```
EIGEN_URL="$(grep -o 'https://storage.googleapis.com/mirror.tensorflow.org/gitlab.com/libeigen/eigen/.*tar.gz' "${BZL_FILE_PATH}")"
```

**Comment by rushabh-v** [comment]
Can I open a PR if you are not interested @thelinuxmaniac ?

**Comment by thelinuxmaniac** [comment]
Did you mean "you are interested"? Yes, please open a PR so that I can send a fix.

**Comment by thelinuxmaniac** [comment]
patch:

```
sed -i "/^EIGEN_URL=/c\EIGEN_URL=\"\$(grep -o 'https://storage.googleapis.com/mirror.tensorflow.org/gitlab.com/libeigen/eigen/.*tar.gz' \"\${BZL_FILE_PATH}\")\"" tensorflow/lite/tools/make/download_dependencies.sh
```

**Comment by rushabh-v** [comment]
>I did not understand what you meant by "Can I open a PR if you are not interested".

I meant that I can open if you are not interested or you don't have time for that. 
But yes you are right, we should ask the maintainers first.

**Comment by thelinuxmaniac** [comment]
@rushabh-v Thank you for offering to help. Let me wait for response from maintainers and I will send a pull request, if required.

**Comment by thelinuxmaniac** [comment]
> I think this was fixed in [0071950](https://github.com/tensorflow/tensorflow/commit/0071950c073eb78e8935ad680f557e11f52b76a0) and [0438fe6](https://github.com/tensorflow/tensorflow/commit/0438fe668efd9c3e3c3d4692f0c4af4f1dff6395), though it is not release in TensorFlow 2.1

closing this issue as it has already been fixed.
